modules/uptime-check/vars.tf

```diff
@@ -1,22 +1,22 @@
-variable "monitoring_project_id" {
+variable monitoring_project_id {
   type        = string
   description = "Project ID to create monitoring resources in"
   default     = "hiiretail-monitoring-prod-6500"
```

Comment:
We need to make sure we have `monitoring_project_id` in project.hcl file in each common repo, otherwise it will use the default value, and then we'll get errors because of the notification channel